cloud/misc/ovirt.py

```diff
@@ -24,7 +24,7 @@
 short_description: oVirt/RHEV platform management
 description:
     - allows you to create new instances, either from scratch or an image, in addition to deleting or stopping instances on the oVirt/RHEV platform
-version_added: "1.4"
+version_added: "2.1"
```

Comment:
I had to change it for my new options because travis ci would fail
otherwise, so i figured i would update the version for the module too.
Wrong call?

> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ansible/ansible-modules-extras/pull/1922/files/fdacc29659540b63660c91d899e48ccad70fceea#r57838818>
>